src/main/java/com/amazon/ion/impl/macro/MacroEvaluatorAsIonReader.kt

```diff
@@ -93,9 +93,9 @@ class MacroEvaluatorAsIonReader(
     }
 
     override fun close() { /* Nothing to do (yet) */ }
-    override fun <T : Any?> asFacet(facetType: Class<T>?): Nothing = TODO("Not supported")
+    override fun <T : Any?> asFacet(facetType: Class<T>?): Nothing? = null
     override fun getDepth(): Int = containerStack.size()
-    override fun getSymbolTable(): SymbolTable = TODO("Not implemented in this abstraction")
+    override fun getSymbolTable(): SymbolTable? = null
```

Comment:
🗺️ Rather than throwing exceptions, these now return `null`, which is allowed by the interface. I don't know why I didn't do that to begin with, but now it means we can use e.g. `_Private_IonSystem.iterate()` on a `MacroEvaluatorAsIonReader` instance.